test.js

```diff
@@ -0,0 +1,2 @@
+console.log("We are coding gods!")
+print("We can do a good job!")
```

Comment:
Well done! I knew the basic idea of how to collaborate in GitHub. 
And have a good night, bros!

Reply:
Good job bro! Have a good night!